### PR TITLE
Add BitBreadcrumb component (#1188)

### LINKED ISSUE #1188: Add Breadcrumb Component
Add **Breadcrumb** Component based on [Microsoft FluentUI Design](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb).

### PR DESCRIPTION
**Pull request by rbasereh**
this closes #1188

### CODE REVIEW

**Comment by rbasereh** [comment]
Add breadcrumb (#1188) #1469
Breadcrumbs used as a navigational aid

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.Fluent.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿.bit-brc-fluent {
+    padding: 16px;
+    list-style: none;
+}
+
+.bit-brc-fluent li{
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.Fluent.scss:6]: plz fix the code formatting issue

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BreadcrumbItem.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        /// If provided, the breadcrumb will be rendered as a link.
+        /// </summary>
+        public string? href { get; set; }
+
+        /// <summary>
+        /// Whether this is the breadcrumb item the user is currently navigated to.
+        /// If true, aria-current="page" will be applied to this breadcrumb item.
+        /// </summary>
+        public bool IsCurrentItem { get; set; }
+        /// <summary>
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BreadcrumbItem.cs:32]: plz add a newline before this line

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+        /// <summary>
+        /// Optional index where overflow items will be collapsed.
+        /// </summary>
+        [Parameter] public byte? OverflowIndex { get; set; }
+        /// <summary>
+        /// Render a custom divider in place of the default chevron >
+        /// </summary>
+        [Parameter] public string DividerAs { get; set; } = "bit-icon--ChevronRight";
+        [Parameter] public string OnRenderOverflowIcon { get; set; } = "bit-icon--More";
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs:34]: plz add an empty line before this line

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs:37]: plz remove these empty lines

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BreadcrumbItem.cs [hunk 5]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.JSInterop;
+namespace Bit.Client.Web.BlazorUI
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/BreadcrumbItem.cs:7]: plz add an empty line before this line

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor [hunk 6]
```diff
@@ -0,0 +1,28 @@
+﻿@namespace Bit.Client.Web.BlazorUI
+@inherits BitComponentBase
+<div @ref="RootElement"
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor:3]: plz add an empty line before this line

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class Breadcrumb
+    {
+        protected override string RootElementClass => "bit-brc";
+        /// <summary>
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs:11]: plz add a newline before this line

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        /// Collection of breadcrumbs to render
+        /// </summary>
+        [Parameter] public List<BreadcrumbItem> Items { get; set; } = new List<BreadcrumbItem>();
+        /// <summary>
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs:15]: plz add a newline before this line

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        /// Aria label for the overflow button.
+        /// </summary>
+        [Parameter] public string? OverflowAriaLabel { get; set; } = "bit-icon--More";
+
+        /// <summary>
+        /// Optional index where overflow items will be collapsed.
+        /// </summary>
+        [Parameter] public byte? OverflowIndex { get; set; }
+        /// <summary>
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Breadcrumb/Breadcrumb.razor.cs:30]: plz add a newline before this line

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**